src/careamics/lvae_training/dataset/vae_dataset.py

```diff
@@ -177,9 +173,9 @@ def __init__(
 
         self._rotation_transform = None
         if self._enable_rotation:
-            raise NotImplementedError(
-                "Augmentation by means of rotation is not supported yet."
-            )
+            # TODO: fix this import
+            import albumentations as A
+
             self._rotation_transform = A.Compose([A.Flip(), A.RandomRotate90()])
```

Comment:
I guess flip/rotate can be safely replaced by existing funcs

Reply:
There is custom logic for flips and rotations in 3D, I'm not sure yet how compatible it is with the existing transforms

Reply:
Are flips and rotations applied in 3D?

I never saw a verification, but PSFs being usually not symmetrical in 3D, this is potentially bad. If data is limiting, that might help, but otherwise that might introduce issues.

We could easily extend the Flip and Rot90 to Z, with the right parameters to turn them off. This would be better than having Albumentations back in.

Reply:
Heard from Ashesh that he is not using z-flips or rotations (which harm the training), so we can just use the CAREamics transformations directly instead of the Albumentations. (at least it should work right away hopefully)

Reply:
PR here #238